diff -arupN lswm/Makefile.in lswm/Makefile.in
--- lswm/Makefile.in	2007-07-28 11:47:03.000000000 -0700
+++ lswm/Makefile.in	2011-10-23 14:59:39.864823230 -0700
@@ -8,7 +8,7 @@ SOURCES = lswm.c
 
 CFLAGS += -I@top_builddir@/libcwiid
 LDFLAGS += -L@top_builddir@/libcwiid
-LDLIBS += -lcwiid
+LDLIBS += -lcwiid -lbluetooth -lrt -lpthread
 INST_DIR = @bindir@
 
 include $(COMMON)/include/app.mak
diff -arupN python/Makefile.in python/Makefile.in
--- python/Makefile.in	2007-08-23 17:21:13.000000000 -0700
+++ python/Makefile.in	2011-10-23 15:20:39.692818079 -0700
@@ -7,7 +7,7 @@ ifdef ROOTDIR
 endif
 
 all:
-	$(PYTHON) setup.py build_ext $(DEBUGFLAGS) -I@top_builddir@/libcwiid -L@top_builddir@/libcwiid -lcwiid
+	$(PYTHON) setup.py build_ext $(DEBUGFLAGS) -I@top_builddir@/libcwiid -L@top_builddir@/libcwiid -lcwiid --rpath ${libdir}
 
 install:
 	$(PYTHON) setup.py install --install-lib=${libdir}/python@PYTHON_VERSION@/site-packages $(SET_ROOT_DIR)
diff -arupN wmdemo/Makefile.in wmdemo/Makefile.in
--- wmdemo/Makefile.in	2007-04-10 19:57:27.000000000 -0700
+++ wmdemo/Makefile.in	2011-10-23 14:58:50.456823432 -0700
@@ -8,7 +8,7 @@ SOURCES = wmdemo.c
 
 CFLAGS += -I@top_builddir@/libcwiid
 LDFLAGS += -L@top_builddir@/libcwiid
-LDLIBS += -lcwiid
+LDLIBS += -lcwiid -lbluetooth -lrt -lpthread
 INST_DIR = @bindir@
 
 include $(COMMON)/include/app.mak
diff -arupN wmgui/Makefile.in wmgui/Makefile.in
--- wmgui/Makefile.in	2007-07-28 11:47:03.000000000 -0700
+++ wmgui/Makefile.in	2011-10-23 14:59:05.696823368 -0700
@@ -8,7 +8,7 @@ SOURCES = main.c interface.c support.c
 
 CFLAGS += @GTK_CFLAGS@ -I@top_builddir@/libcwiid
 LDFLAGS += -L@top_builddir@/libcwiid
-LDLIBS += @GTK_LIBS@ -lm -lcwiid
+LDLIBS += @GTK_LIBS@ -lm -lcwiid -lbluetooth -lrt -lpthread
 INST_DIR = @bindir@
 
 include $(COMMON)/include/app.mak
diff -arupN wminput/Makefile.in wminput/Makefile.in
--- wminput/Makefile.in	2008-06-05 07:09:16.000000000 -0700
+++ wminput/Makefile.in	2011-10-23 14:59:15.060823331 -0700
@@ -14,7 +14,7 @@ CFLAGS += -I@top_builddir@/libcwiid \
 	-DWMINPUT_CONFIG_DIR=\"$(WMINPUT_CONFIG_DIR)\" \
 	-DCWIID_PLUGINS_DIR=\"$(CWIID_PLUGINS_DIR)\"
 
-LDLIBS += -lcwiid -ldl
+LDLIBS += -lcwiid -ldl -lbluetooth -lrt -lpthread
 LDFLAGS += -L@top_builddir@/libcwiid -rdynamic
 INST_DIR = @bindir@
 
